app/models/metric/capture.rb

```diff
@@ -43,12 +43,13 @@ def self.alert_capture_threshold(target)
                 ::Settings.performance.capture_threshold_with_alerts.default)
   end
 
-  def self.perf_capture_timer(zone = nil)
+  def self.perf_capture_timer(ems_id = nil)
```

Comment:
Are there any places that call `perf_capture_timer` without an argument?  If this is just for dev ease-of-use I'd rather require an ems so we don't need the `capture_targets(ems || zone)` later on

Reply:
We do know that `perf_capture_timer` will not be called with a zone. Passing an AR object over the queue is not even possible, so that case we're in the clear.

`performance_capture_gap` takes in a `zone_id` [[ref]](https://github.com/kbrock/manageiq/blob/master/app/models/metric/capture.rb#L68). It then calls into `capture_targets(zone)`.

Since the ui prompts the user for a `zone`, we can't completely swap `performance_capture_gap` over to `ems_id` without some ui change and being backwards compatible for the foreseen future. But we can change gap to take an ems_id or zone_id, and have call into `capture_targets` with only an ems.

Also, we probably have existing `perf_capture_timer` messages in the queue with no parameter. I had supported these, but due to not wanting to run 2 collectors at the same time, we probably want to drop these duplicate timer messages. The 2 different signatures will not properly de-dup and possibly allow multiple to be run at once.

Reply:
reverted back to `capture_targets(zone)` so it can be called by `performance_capture_gap`
Instead we'll call `capture_ems_targets(ems)` directly - and phase that method out